src/biogeochem/DUSTMod.F90

```diff
@@ -235,13 +377,38 @@ subroutine DustEmission (bounds, &
     real(r8) :: sumwt(bounds%begl:bounds%endl) ! sum of weights
     logical  :: found                          ! temporary for error check
     integer  :: index
+
+    real(r8) :: tmp2                ! calculates the dry fluid threshold using Shao and Lu (2000) scheme;
+                                    ! replace the tmp1 (Iversen and White, 1982) that was passed from Dustini to DustEmission;
+                                    ! tmp2 will be calculated here
+    real(r8) :: wnd_frc_thr_slt_std ! [m/s] The soil threshold friction speed at standard air density (1.2250 kg/m3)
+    real(r8) :: frag_expt           ! fragmentation exponent
+    real(r8) :: wnd_frc_thr_slt_it  ! [m/s] created for impact threshold friction velocity
+    real(r8) :: wnd_frc_thr_slt     ! [m/s] used for wet fluid threshold friction velocity
+    real(r8) :: K_length            ! [dimless] normalized mean interobstacle distance, or called gap length (Okin, 2008)
+    real(r8) :: bare_frc            ! LUH2 bare soil land cover fraction
+    real(r8) :: veg_frc             ! LUH2 natural vegetation + crop land cover fraction
     !    
     ! constants
     !
     real(r8), parameter :: cst_slt = 2.61_r8           ! [frc] Saltation constant
     real(r8), parameter :: flx_mss_fdg_fct = 5.0e-4_r8 ! [frc] Empir. mass flx tuning eflx_lh_vegt
-    real(r8), parameter :: vai_mbl_thr = 0.3_r8        ! [m2 m-2] VAI threshold quenching dust mobilization
```

Comment:
I suddenly recalled that vai_mbl_thr = 0.3_r8 is for Zender's scheme, so we can't delete it here. We will use vai_mbl_thr = 0.3_r8 for Zender and vai_mbl_thr = 1.0_r8 for Leung.